ci-operator/jobs/infra-periodics.yaml

```diff
@@ -608,6 +608,105 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  labels:
+    ci.openshift.io/role: infra
+  decorate: true
+  interval: 30m
```

Comment:
Given that we apparently still have problems with token consumption, should we run this less often? Daily perhaps?